python/py-userpath/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-userpath
+version             1.3.0
+categories-append   devel
+license             {MIT Apache}
+maintainers         @0az openmaintainer
+supported_archs     noarch
+platforms           darwin
+
+description         A tool and library for adding locations to user PATH
+long_description    ${description}
+
+homepage            https://github.com/mitsuhiko/click
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            userpath-${version}
+
+checksums           rmd160  d3997f6d4d9502637e173d9f31006b8e27ec5361 \
+                    sha256  dd0b8ba650732c614c5e6b709e51be321c596566422d99a443d545401a965468 \
+                    size    19056
+
+python.versions     27 36 37 38
```

Comment:
it appears that the ```27``` version isn't needed for any of the currently available ports; in that case I'd recommend not to add this version anymore.